bin/Phantom/cards/examples/phantomProduction_example.cfg

```diff
@@ -0,0 +1,177 @@
+# ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ---- #
+#                                                                                        #
+#           template cfg file for the usage of the phantom generator in CMS              #
+#                                                                                        #
+# see https://twiki.cern.ch/twiki/bin/view/CMS/PhantomGeneratorCMS for more details      #
+#                                                                                        #
+# ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ----  ---- ---- #
+
+
+[general]
+# where the zipped version of the compiled phantom code can be found
+# and where the gridpacks will be created
+
+# CMSSW environment for the gridpack creation
+CMSSW = CMSSW_9_2_6_patch1
+
+# phantom version to be used
+package = https://cms-project-generators.web.cern.ch/cms-project-generators/phantom/phantom_1_3_slc6_amd64_gcc530.tar.gz
```

Comment:
I suggest to use the existing convention, cfr
https://cms-project-generators.web.cern.ch/cms-project-generators/slc6_amd64_gcc481/powheg/V2.0/src/
can you change to 
https://cms-project-generators.web.cern.ch/cms-project-generators/slc6_amd64_gcc530/phantom/V1.3/phantom_1_3_slc6_amd64_gcc530.tar.gz
Also, can you rename phantom_1_3.tar.gz to phantom_1_3_src.tar.gz to make it clearer that is the source tarball?

Reply:
Ciao Luca,
I prefer not to adhere to that convention, since it's not given that I will compile a version for slc6_amd64_gcc481, while it's sure there will be a phantom generator, so it makes more sense to me to have a phantom folder containing the precompiled versions which exist in CMS

Reply:
The source name is the one I get from the authors, this is why I did not change it

Reply:
ok for the folder convention. if you want to keep the source tarball with the same name, please put it under a folder called "src"

Reply:
ok, let me test the modifications, there's now two folders: source and compiled.

Reply:
it works, PR updated